core/src/main/java/org/apache/accumulo/core/iterators/Combiner.java

```diff
@@ -130,7 +131,7 @@ public Value next() {
         source.next();
         hasNext = _hasNext();
       } catch (IOException e) {
-        throw new RuntimeException(e);
+        throw new UncheckedIOException(e);
```

Comment:
This change does not contribute to the correctness of this PR in any way, just though it was nice to do since I noticed while debugging.